closed/make/modules/openjceplus/Copy.gmk

```diff
@@ -0,0 +1,49 @@
+# ===========================================================================
+# (c) Copyright IBM Corp. 2023, 2023 All Rights Reserved
+# ===========================================================================
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# IBM designates this particular file as subject to the "Classpath" exception
+# as provided by IBM in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+# ===========================================================================
+
+include $(TOPDIR)/closed/CopySupport.gmk
+
+ifeq (true,$(BUILD_OPENJCEPLUS))
+  # Copy OpenJCEPlus native libraries
+  $(eval $(call SetupCopyFiles, COPY_JGSKIT_LIBS, \
+      SRC := $(OPENJCEPLUS_TOPDIR)/target, \
+      FILES := $(filter %.dll %.so %.x, $(call FindFiles, $(OPENJCEPLUS_TOPDIR)/target)), \
+      FLATTEN := true, \
+      DEST := $(LIB_DST_DIR), \
+  ))
+
+  TARGETS += $(COPY_JGSKIT_LIBS)
+
+  # Bundle GSKIT library
+  OCK_DIR := $(TOPDIR)/OpenJCEPlus/ock/jgsk_sdk/lib64
+  ifeq ($(call isTargetOs, windows), true)
+    OCK_SUB_DIR := modules_cmds
+  else
+    OCK_SUB_DIR := modules_libs
+  endif
+
```

Comment:
I think the makefiles should restrict themselves to a single root name (suggest `OPENJCEPLUS`) rather than also use `GSKIT`, `JGSKIT`, and `OCK`.